addon/components/bs-modal.js

```diff
@@ -473,7 +473,7 @@ export default class Modal extends Component {
     if (!this._isOpen) {
       return;
     }
-    this.isOpen = false;
+    this._isOpen = false;
```

Comment:
If I didn't missed something, this was the only place, in which `isOpen` was actually set. So we wouldn't need `@localCopy` on it anymore. It would be just an alias to `this.args.isOpen`.

If the big could be resolved by _not_ setting `isOpen`, would actually be inline with my findings in https://github.com/kaliber5/ember-bootstrap/issues/1386#issuecomment-756933816.

Reply:
I've refactored out the @localCopy. I've tried to set the value to `this.args.open`, however it seems like that breaks more than it fixes. Accepting open as an argument with a default value of `true` does not seem to break any tests. Any calls to `this.isOpen` without the underscore has been changed to open.

Reply:
Glad to see that we could actually drop `@localCopy` decorator. Less complexity is always good. :smile: 

Your changes look very good. I'm just a little bit confused why you haven't stuck with `isOpen` name. The `is` prefix is a helpful indicator that a property is a boolean in my opinion. In this special case I also fear that a property `open` might be confused with the `show` method.

Would it be okay for you to change the name back to `isOpen`? Not the implementation. That one is totally fine. I'm only talking about the naming.